src/main/java/gregtech/common/covers/detector/CoverDetectorEnergyAdvanced.java

```diff
@@ -88,72 +102,93 @@ private void compareValue(long value, long maxValue, long minValue) {
         }
     }
 
+    private void compareValue(float value, long maxValue, long minValue) {
+        if (value >= maxValue) {
+            this.outputAmount = inverted ? 15 : 0;
+        } else if (value <= minValue) {
+            this.outputAmount = inverted ? 0 : 15;
+        }
+    }
```

Comment:
I recall having a function which does this already. I believe it is located in GTUility. If not, disregard this comment.

Reply:
The method in `GTUtility` outputs a ratio between 0-15 between `min` and `max`, values less than `min` or more than `max` outputs 0 or 15 respectively, and does not have latching functionality built in. This method is slightly different, with latching functionality and only outputting 0 or 15. This method could be moved to `GTUtility`, if need be.